exercises/strings/rotN/rotN.js

```diff
@@ -15,15 +15,37 @@
  * @returns {string} A lowercase copy of the input string
  */
 
-function rotN(string) {
+function rotN(string, n) {
   // This is your job. :)
+  var i;
+  var new_string = '';  
+  for (i=0; i<string.length; i++){
+    var char = string.charCodeAt(i);
+    // lower case letter
+    if ((char >= 97) && (char <= 122)){
```

Comment:
1. You can compare strings directly in JavaScript, e.g., `char > 'a'`.
1. Rather than encoding "magic numbers" you can do something like `'a'.charCodeAt(0)`.

These will make it more obvious to any reader what's going on (and eliminate the possibility of off-by-one errors).